.eslintrc.js

```diff
@@ -355,5 +312,13 @@ module.exports = {
         'no-await-in-loop': 0,
       },
     },
+    {
+      // TODO: Remove this overrides after extending `plugin:@typescript-eslint/recommended` config
+      files: ['*.ts', '*.tsx'],
+      rules: {
+        'no-redeclare': 'off', // ts(2451)
+        'no-undef': 'off', // ts(2304)
```

Comment:
the following rules are turned off in `plugin:@typescript-eslint/recommended` config as they already catched by `tsc`